test/gtest/ucp/test_ucp_perf.cc

```diff
@@ -357,6 +357,17 @@ UCS_TEST_SKIP_COND_P(test_ucp_perf, envelope, has_transport("self"))
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_perf)
 
+class test_ucp_loopback : public test_ucp_perf {};
+
+UCS_TEST_P(test_ucp_loopback, envelope)
+{
+    test_spec test = tests[get_variant_value(VARIANT_TEST_TYPE)];
+
+    run_test(test, UCX_PERF_TEST_FLAG_LOOPBACK, true, "", "");
```

Comment:
pls also add execution of "ucx_perftest" in loopack mode to test_jenkins.sh (currently it's being run as cleint/server and also as MPI application)
it's enough to just have smoke-test with "self" transport for now

Reply:
unhandled - we need some execution of ucx_perftest loobpoack in CI to make sure it's working, not only from gtest